### PR TITLE
Add region ignore with gmask

### PR DESCRIPTION
**Pull request by OneLiteFeather**
## Overview
Some players do not know how to exclude a part of their build so that only a part is changed
This was raised in https://github.com/IntellectualSites/documentation/issues/67

## Description
Add a step-by-step tutorial how to do it

```[tasklist]
### Submitter Checklist
- [x] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [x] Ensure that the pull request title represents the desired changelog entry.
- [x] New public fields and methods are annotated with `@since TODO`.
- [x] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md).
```

### CODE REVIEW

**Comment by NotMyFault** [comment]
Please compress the images, i.e. https://compressor.io/

**Comment by OneLiteFeather** [comment]
> Please compress the images, i.e. https://compressor.io/

Done, hopefully we can minimize the storage footprint with that

File: fastasyncworldedit/advanced-commands/selection.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Introduction
+
+In some cases you want to exclude regions because you don't want to edit them, 
+for example when the unwanted part is inside another wanted part.
+In this example we use the [wesui plugin of kennytv](https://www.spigotmc.org/resources/worldeditsui-visualize-your-selection.60726/) (is optional, but can help to understand each step) to visualize the selections.
```
Comment (SirYwell) [inline fastasyncworldedit/advanced-commands/selection.md:5]: It's enough to only mention the name, but it should be `WorldEditSUI`.

Comment (OneLiteFeather) [reply fastasyncworldedit/advanced-commands/selection.md:5]: Can I leave the link in there? There are 2 plugins that are similar but that's more maintained/active and it is from a known paper developer.

Comment (SirYwell) [reply fastasyncworldedit/advanced-commands/selection.md:5]: Yes, the link is fine.

**Review by NotMyFault — APPROVED** [review]

**Comment by RedstoneFuture** [comment]
I think this is very rudimentary. We actually show separate documentation in the planned mask instructions. These instructions do not fit into the plugin documentation in the "selection.md" in terms of style and execution.

**Comment by OneLiteFeather** [comment]
> EDIT: Now I see "/advanced-commands" is a new category, okay. Perhaps we could also call it "Start-up Tutorial".

I had the idea to name it this way to allow adding the special and more in depth commands from the community which would be lost in the start up tutorials. For example, there are very advance / special commands on the guide from zeranny and eztak I have allowance to add and mention, but very helpful for advanced builders.
This documentation on the PR is also something you rarely need because it is special.

**Comment by TheMeinerLP** [comment]
Whats the status here ?

**Comment by NotMyFault** [comment]
@SirYwell Do you want to take another look, please?

**Review by SirYwell — APPROVED** [review]

**Pull request merged**